content/blog/sprint-review-2103.md

```diff
@@ -0,0 +1,45 @@
+Title: Wazo Platform 21.03 Released
+Date: 2021-03-08 16:00
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2103
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 21.03 release.
+
+## New features in this release
+
+* **Call Recording**: it is now possible to record calls that are coming from and going to a queue or a group.
+* **Call Recording**: There is a new API to delete multiple recordings, useful to purge recordings automatically.
```

Comment:
**Suggestion :** `~~There is a new~~ API to delete multiple recordings, useful to purge recordings automatically.`

We are already n the new feature section 😃